web/src/types/modules/setting.d.ts

```diff
@@ -10,6 +10,8 @@ interface Setting {
 interface LocalSetting {
   enableFoldMemo: boolean;
   enableDoubleClickEditing: boolean;
+  dailyReviewTimeOffsetHourValue: number;
+  dailyReviewTimeOffsetMinuteValue: number;
```

Comment:
Since the time zone is divided according to the hour, so I think saving the hours should be enough. So we don't have to save minute.

Reply:
Changed time offset to only include hour